octobot_evaluators/evaluator/strategy_evaluator.py

```diff
@@ -57,6 +128,201 @@ async def matrix_callback(self,
+                    time_frame: value
+                }
+
+    def _get_exchange_current_time(self, exchange_name, matrix_id):
+        try:
+            from octobot_trading.api.exchange import get_exchange_current_time, \
+                get_exchange_manager_from_exchange_name_and_id, get_exchange_id_from_matrix_id
+            exchange_manager = get_exchange_manager_from_exchange_name_and_id(
+                exchange_name,
+                get_exchange_id_from_matrix_id(exchange_name, matrix_id)
+            )
+            return get_exchange_current_time(exchange_manager)
+        except ImportError:
+            self.logger.error("Strategy requires OctoBot-Trading package installed")
+
+    async def evaluators_callback(self,
+                                  matrix_id,
+                                  evaluator_name,
+                                  evaluator_type,
+                                  exchange_name,
+                                  cryptocurrency,
+                                  symbol,
+                                  time_frame,
+                                  data):
+        # Used to communicate between evaluators
+        if data[EVALUATOR_CHANNEL_DATA_ACTION] == RESET_EVALUATION:
+            for time_frame in data[EVALUATOR_CHANNEL_DATA_TIME_FRAMES]:
+                self._set_last_evaluation_time(exchange_name, EvaluatorMatrixTypes.TA.value, cryptocurrency, symbol,
+                                               time_frame.value, None)
+
+    def get_evaluations_by_evaluator(self,
+                                     matrix_id,
+                                     exchange_name=None,
+                                     tentacle_type=None,
+                                     cryptocurrency=None,
+                                     symbol=None,
+                                     time_frame=None,
+                                     allow_missing=True,
+                                     allowed_values=None) -> dict:
+        evaluator_nodes = get_node_children_by_names_at_path(matrix_id,
+                                                             get_tentacle_path(exchange_name=exchange_name,
+                                                                               tentacle_type=tentacle_type))
+        evaluations_by_evaluator = {}
+        for evaluator_name, node in evaluator_nodes.items():
+            evaluation = get_tentacles_value_nodes(matrix_id, [node], cryptocurrency=cryptocurrency,
+                                                   symbol=symbol, time_frame=time_frame)
+            if len(evaluation) > 1:
+                self.logger.warning("More than one evaluation corresponding to the given tentacle filter, "
+                                    "this means there is an issue in this methods given arguments")
+            else:
+                if evaluation:
+                    eval_value = evaluation[0].node_value
```

Comment:
As the previous comment about `matrix_manager`, I think we shouldn't use matrix low level attributes here.